Open-Rival/src/Unit.cpp

```diff
@@ -4,37 +4,39 @@
 namespace Rival {
 
     Unit::Unit(UnitType type) :
-        type(type) {}
+        type(type),
+        animation(type) {}
 
     void Unit::addedToWorld(
             int id, int player, int x, int y, Facing facing) {
         this->id = id;
         this->player = player;
         this->x = x;
         this->y = y;
-        this->facing = facing;
+        animation.setFacing( facing );
```

Comment:
We are introducing some inconsistency in the spacing around parentheses (here and elsewhere). If your IDE is fighting you on this, it may be possible to change some formatting settings. Or if it's just habit, then we need to find a compromise 😜